### ISSUE TITLE
Add Actors Replace

### ISSUE THREAD

**Issue by drraymond27** [DirtyRacer1337/PhoenixAdult.bundle#64]
Add Actors Replace

**Comment by drraymond27** [comment]
Anyone?

**Comment by DirtyRacer1337** [comment]
Actor with one word in alias should be added per studio replace, also I noticed some of your request already exists, check on latest.

You can do it by yourself in [Contents/Code/PAdatabaseActors.py](https://github.com/DirtyRacer1337/PhoenixAdult.bundle/blob/master/Contents/Code/PAdatabaseActors.py), but don't forget to share

or write correct data in format below and I add it

Format
---
- Actor name: alias1, alias2
- Actor name 2: alias1, alias2
---
- Studio name:
  - Actor name: alias1, alias2
  - Actor name 2: alias1, alias2
- Studio name 2:
  - Actor name: alias1, alias2
  - Actor name 2: alias1, alias2

**Comment by drraymond27** [comment]
OK, I'll try
Thanks

**Comment by drraymond27** [comment]
@DirtyRacer1337 I added some Actors Names, I hope I did it well

**Comment by DirtyRacer1337** [comment]
@drraymond27 I fixed some errors, 0d8d31b1c2f0ba9223cc2d7075976d8475c2785d

**Comment by drraymond27** [comment]
@DirtyRacer1337 In Porndoe has a Actress with the name Selena Gomez (her real name is Karen), whose cause conflict with the real Selena Gomez in others Libraries (that have children contents), and I think that we change the name to don't match with the real Selena Gomez

**Comment by DirtyRacer1337** [comment]
@drraymond27 I leaved it, for better solution. At beggining I wanted say that it's not a problem, you don't have children contents and xxx in same library, but they use one image.
She doesn't have another alias.
https://www.freeones.ru/selena-gomez-p/profile
https://www.indexxx.com/m/selena-gomez

So I came to this 3d53b26697d80ea76cbeb45531fa5093dbeeb504, Now name looks same, but they should be not crossed.
